### PR TITLE
chore: update env variable descriptions

### LINKED ISSUE #2271: Improve description of environment variables
in the docs

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- mainly updating docs
- backend: deprecate env variable PUBLIC_HOST in favor for OPEN_PAYMENTS_URL
- auth: removed redundant PORT in favor for AUTH_PORT

## Context

- fixes #2271 

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

File: packages/documentation/src/content/docs/integration/deployment.md [hunk 1]
```diff
@@ -114,82 +113,80 @@ Now, the Admin UI can be found on localhost:3010.
...
-| `WITHDRAWAL_THROTTLE_DELAY`                           | `undefined`                                                 | delay in withdrawal processing                                                                                                                         |
+| Variable                                              | Default                                                     | Description                                                                                                                                                                                                |
+| ----------------------------------------------------- | ----------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
```
Comment (raducristianpopa) [inline packages/documentation/src/content/docs/integration/deployment.md:117]: Table is broken in the deployment preview.

```suggestion
| Variable                                              | Default                                                     | Description | 
| ----------------------------------------------------- | ----------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

File: packages/auth/src/config/app.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const Config = {
   identityServerSecret: envString('IDENTITY_SERVER_SECRET', 'replace-me'),
   authServerDomain: envString(
     'AUTH_SERVER_DOMAIN',
-    `http://localhost:${envInt('PORT', 3006)}`
+    `http://localhost:${envInt('AUTH_PORT', 3006)}`
```
Comment (raducristianpopa) [inline packages/auth/src/config/app.ts:45]: Will probably need to add the `breaking` label because of this?

Comment (sabineschaller) [reply packages/auth/src/config/app.ts:45]: You are right.

**Comment by sabineschaller** [comment]
Table still broken...

File: packages/documentation/src/content/docs/integration/deployment.md (hunk 1 above)
Comment (raducristianpopa) [inline packages/documentation/src/content/docs/integration/deployment.md:117]:
```suggestion
| Variable                                              | Default                                                     | Description                                                                                                                                                                                                |
| ----------------------------------------------------- | ----------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

File: packages/documentation/src/content/docs/integration/deployment.md [hunk 3]
```diff
@@ -114,82 +113,80 @@ Now, the Admin UI can be found on localhost:3010.
...
+| `WALLET_ADDRESS_DEACTIVATION_PAYMENT_GRACE_PERIOD_MS` | `86400000`                                                  | time in milliseconds into the future to set expiration of open incoming payments when deactivating wallet address. Default: 1 days                                                                         |
+| `WALLET_ADDRESS_LOOKUP_TIMEOUT_MS`                    | `1500`                                                      | time in milliseconds the ASE has to create a missing wallet address until timeout                                                                                                                          |
+| `WALLET_ADDRESS_POLLING_FREQUENCY_MS`                 | `100`                                                       | frequency of polling while waiting for ASE to create a missing wallet address                                                                                                                              |     |
```
Comment (raducristianpopa) [inline packages/documentation/src/content/docs/integration/deployment.md:158]:
```suggestion
| `WALLET_ADDRESS_POLLING_FREQUENCY_MS`                 | `100`                                                       | frequency of polling while waiting for ASE to create a missing wallet address                                                                                                                              | 
```

**Review by raducristianpopa — COMMENTED** [review]
This should fix it? 🤔

**Review by raducristianpopa — APPROVED** [review]

File: packages/backend/src/config/app.ts [hunk 4]
```diff
@@ -33,9 +33,9 @@ dotenv.config({
 export const Config = {
   logLevel: envString('LOG_LEVEL', 'info'),
   // publicHost is for open payments URLs.
-  publicHost: envString('PUBLIC_HOST', 'http://127.0.0.1:3001'),
+  publicHost: envString('OPEN_PAYMENTS_URL', 'http://127.0.0.1:3000'),
```
Comment (mkurapov) [inline packages/backend/src/config/app.ts:36]: can we remove `publicHost` altogether?

File: packages/documentation/src/content/docs/integration/deployment.md [hunk 5]
```diff
@@ -114,82 +113,80 @@ Now, the Admin UI can be found on localhost:3010.
...
+| `SIGNATURE_SECRET`                                    | `undefined`                                                 | secret to generate request header signatures for webhook event requests                                                                                                                                    |
+| `SIGNATURE_VERSION`                                   | `1`                                                         | version number to generate request header signatures for webhook event requests                                                                                                                            |
+| `SLIPPAGE`                                            | `0.01`                                                      | accepted quote fluctuation, default 1%                                                                                                                                                                     |
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/integration/deployment.md:150]:
```suggestion
| `SLIPPAGE`                                            | `0.01`                                                      | accepted ILP quote fluctuation, default 1%                                                                                                                                                                     |
```

File: packages/documentation/src/content/docs/integration/deployment.md [hunk 6]
```diff
@@ -114,82 +113,80 @@ Now, the Admin UI can be found on localhost:3010.
...
+| `OUTGOING_PAYMENT_WORKERS`                            | `4`                                                         | number of workers processing outgoing payment requests                                                                                                                                                     |
+| `OUTGOING_PAYMENT_WORKER_IDLE`                        | `200`                                                       | time in milliseconds that OUTGOING_PAYMENT_WORKERS will wait until they check an empty outgoing payment request queue again                                                                                |
+| `PRIVATE_KEY_FILE`                                    | `undefined`                                                 | this Rafiki instance's client private key                                                                                                                                                                  |
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/integration/deployment.md:142]:
```suggestion
| `PRIVATE_KEY_FILE`                                    | `undefined`                                                 | the path to this Rafiki instance's client private key                                                                                                                                                                  |
```

**Review by njlie — APPROVED** [review]

**Review by njlie — APPROVED** [review]

**Pull request merged**